db/migrate/20181008141711_add_locale_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddLocaleToUser < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :locale, :string, null: false, default: 'en'
```

Comment:
You forgot to validate this in the user model 😬 (Sorry i did not notice this before)